### ISSUE TITLE
Syncing Issue Windows 10 Simplenote version 1.1.7

### ISSUE THREAD

**Issue by AMcGrady** [Automattic/simplenote-electron#825]
From a User:
I believe I've identified the problem.

In the latest version of the PC app, whichever note first opens when you start the app doesn't get updated until you click on another note and then back again. This forces the note to update. If you make changes to the note before forcing the update, you may end up overwriting changes from another device or causing other weird app behavior.

#### Steps to reproduce
1. Open Note
2. Make changes to it before clicking on another note

#### What I expected
Notes to sync in Cloud Automatically

#### What happened instead
If you make changes to the note before forcing the update, you may end up overwriting changes from another device or causing other weird app behavior.

#### OS version
Windows 10 
Simplenote version 1.1.7

**Comment by djdany01** [comment]
Same problem here.
I use 3 different SimpleNote apps, 2 in Windows 10 and 1 in Android.
Android syncs perfect but windows 10 version no.

**OS version**  

Windows 10
Simplenote version 1.1.7

**Comment by Lochlaer** [comment]
A user submitted a support ticket (#1603901-zen) about a similar syncing issue. Basically, he has a pinned note in his Windows Simplenote app, and one day it disappeared. He logged into the web app as well as the app on two different Android devices, and he was able to see the note on all of those. He made a change in the note on the web app, and then reopened Simplenote on his Windows machine. Voila -- the note was back, this time with the change he'd just made on the web app. 

Simplenote version: v1.2.1
OS: Windows 10 Home, v1803, build 17134.407. 64-bit system

**Comment by mirka** [comment]
I'm unable to reproduce the originally reported issue, neither in v1.1.7 or the current v1.3.3.

The exact steps I'm taking are:

1. Log in to the same account on https://app.simplenote.com and the Windows app.
1. On Windows, make sure that the note list sorting is set to "Last modified".
1. Quit the Windows app.
1. On the web app, change the first line of the first note.
1. Launch the Windows app.

The Windows app launches with the first note selected, and it is quickly updated to reflect the changes made on the web app.

Let me know if there's something else I need to do to reproduce, or if the issue has gone away.

**Comment by mirka** [comment]
I'm going to close this for now, since the original issue was unreproducible and we have some more sync fixes in v1.3.4. Please feel free to post a new issue with details if you still see sync issues in the latest version.

**Comment by bestann** [comment]
Today I editied a note (very important and large) on Windows 10.
During it I opened Simplenote on Android. At first it saw my note but then it dissapeared and another I saw anothe note duplicated. And on Windows also another note duplicated. I workec at that note for a week!!! And its' impossible to restore it's older version because Simplenote has deleted it at all. So maybe you'd add audomatic everyday archiving of notes and saving several  copyies of every note. It's impossible to work with your program.
